### PR TITLE
Add udpate strategy p2-rctl command

### PR DESCRIPTION
**Pull request by spenceral**

### CODE REVIEW

File: pkg/store/consul/rcstore/consul_store.go [hunk 1]
```diff
@@ -663,6 +663,14 @@ func (s *ConsulStore) UpdateManifest(id fields.ID, man manifest.Manifest) error
 	return s.retryMutate(id, manifestUpdater)
 }
 
+func (s *ConsulStore) UpdateStrategy(id fields.ID, strategy fields.Strategy) error {
+	strategyUpdater := func(rc fields.RC) (fields.RC, error) {
```
Comment (mpuncel) [inline pkg/store/consul/rcstore/consul_store.go]: maybe here you should check that you got a real strategy

Comment (mpuncel) [reply pkg/store/consul/rcstore/consul_store.go]: although I didn't do that for pod clusters either so maybe i'm a hypocrite :P

Comment (spenceral) [reply pkg/store/consul/rcstore/consul_store.go]: 👍

Comment (rudle) [reply pkg/store/consul/rcstore/consul_store.go]: didn't you? https://github.com/square/p2/pull/954/files#diff-21e734f407a99256a47190a2322142b1R106

Comment (mpuncel) [reply pkg/store/consul/rcstore/consul_store.go]: ah yeah i meant i didn't do it in a store function, but I also didn't write a store function I just used `MutatePC()`.

File: pkg/store/consul/rcstore/consul_store.go [hunk 2]
```diff
@@ -663,6 +663,18 @@ func (s *ConsulStore) UpdateManifest(id fields.ID, man manifest.Manifest) error
 	return s.retryMutate(id, manifestUpdater)
 }
 
+func (s *ConsulStore) UpdateStrategy(id fields.ID, strategy fields.Strategy) error {
+	if strategy != fields.DynamicStrategy && strategy != fields.StaticStrategy {
```
Comment (spenceral) [inline pkg/store/consul/rcstore/consul_store.go]: checked

**Pull request merged**